tslint.json

```diff
@@ -32,7 +33,15 @@
     "max-classes-per-file": [true, 3],
     "no-var-requires": false,
     "member-ordering": false,
-    "no-floating-promises": true
+    "no-floating-promises": true,
+    "forin": false,
```

Comment:
The rule should stay as is or be set to true if it's not a default.
It's not ideal to skip filtering, even in TS.

See https://palantir.github.io/tslint/rules/forin/